server/services/locations_service_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package services
+
+import (
+	"context"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	locpb "google.golang.org/genproto/googleapis/cloud/location"
+	"google.golang.org/protobuf/proto"
+)
+
+func TestListLocations(t *testing.T) {
+	s := NewLocationsServer()
+	foo := "projects/foo"
```

Comment:
nit: why not s/`foo`/`name`/ or s/`foo`/`locName`/ (for the variable name)?

Reply:
I went with `name`. Done!